osu.Game/Online/API/Requests/SearchBeatmapSetsResponse.cs

```diff
@@ -2,12 +2,20 @@
 // See the LICENCE file in the repository root for full licence text.
 
 using System.Collections.Generic;
+using Newtonsoft.Json;
 using osu.Game.Online.API.Requests.Responses;
 
 namespace osu.Game.Online.API.Requests
 {
     public class SearchBeatmapSetsResponse : ResponseWithCursor
     {
+        [JsonProperty("beatmapsets")]
         public IEnumerable<APIBeatmapSet> BeatmapSets;
+
+        [JsonProperty("error")]
+        public string Error;
```

Comment:
As per the other comment, should be removed, as the contents of this will never be relevant if the response is successful.